uk.co.compucorp.civicrm.tasksassignments/scss/civitasks.scss

```diff
@@ -1,142 +1,146 @@
-  @import "civihr/partials/wells";
-
-  @import "civihr/modules/ie";
+  @import 'civihr/partials/badge';
+  @import 'civihr/partials/breadcrumb';
+  @import 'civihr/partials/buttons';
+  @import 'civihr/partials/collapse';
+  @import 'civihr/partials/dropdowns';
+  @import 'civihr/partials/forms';
+  @import 'civihr/partials/iframe';
+  @import 'civihr/partials/modals';
+  @import 'civihr/partials/page-contact';
+  @import 'civihr/partials/pagination';
+  @import 'civihr/partials/panels';
+  @import 'civihr/partials/tables';
+  @import 'civihr/partials/tooltip';
+  @import 'civihr/partials/wells';
+
+  @import 'civihr/modules/ie';
 
   // CiviHR CiviTasks Ext. Custom
   // ==========================================================================
-  @import "civitasks/modules/variables";
-  @import "civitasks/modules/bootstrap-calendar";
-  @import "civitasks/modules/xeditable";
-  @import "civitasks/modules/ui-bootstrap";
+  @import 'civitasks/modules/variables';
+  @import 'civitasks/modules/bootstrap-calendar';
+  @import 'civitasks/modules/xeditable';
+  @import 'civitasks/modules/ui-bootstrap';
 
-  @import "civitasks/partials/global";
+  @import 'civitasks/partials/global';
   @include global;
-  @import "civitasks/partials/breadcrumb";
-  @import "civitasks/partials/buttons";
-  @import "civitasks/partials/calendar";
-  @import "civitasks/partials/dashboard";
-  @import "civitasks/partials/documents";
-  @import "civitasks/partials/filters";
-  @import "civitasks/partials/forms";
-  @import "civitasks/partials/iframe";
-  @import "civitasks/partials/lists";
-  @import "civitasks/partials/navs";
-  @import "civitasks/partials/page-contact";
-  @import "civitasks/partials/page-settings";
-  @import "civitasks/partials/sidebar";
-  @import "civitasks/partials/spinner";
-  @import "civitasks/partials/tables";
-  @import "civitasks/partials/editable";
-  @import "civitasks/partials/custom-select";
-  @import "civitasks/partials/nav-tabs";
-
-  @import "civitasks/modules/ie";
-  @import "civitasks/modules/civihr-ui-select";
+  @import 'civitasks/partials/breadcrumb';
+  @import 'civitasks/partials/buttons';
+  @import 'civitasks/partials/calendar';
+  @import 'civitasks/partials/dashboard';
+  @import 'civitasks/partials/documents';
+  @import 'civitasks/partials/filters';
+  @import 'civitasks/partials/forms';
+  @import 'civitasks/partials/iframe';
+  @import 'civitasks/partials/lists';
+  @import 'civitasks/partials/navs';
+  @import 'civitasks/partials/page-contact';
+  @import 'civitasks/partials/page-settings';
+  @import 'civitasks/partials/sidebar';
+  @import 'civitasks/partials/spinner';
+  @import 'civitasks/partials/tables';
+  @import 'civitasks/partials/editable';
+  @import 'civitasks/partials/custom-select';
+  @import 'civitasks/partials/nav-tabs';
+
+  @import 'civitasks/modules/ie';
+  @import 'civitasks/modules/civihr-ui-select';
+
+  @import '../../../org.civicrm.shoreditch/scss/bootstrap/overrides/variables';
```

Comment:
use `SCSSROOT`

Reply:
T&A has a compass based build system and SCSSROOT is not integrated yet, so did not add that.